Source/JavaScriptCore/runtime/CommonSlowPaths.cpp

```diff
@@ -923,6 +923,7 @@ JSC_DEFINE_COMMON_SLOW_PATH(slow_path_enter)
 {
     BEGIN();
     Heap::heap(codeBlock)->writeBarrier(codeBlock);
+    GET(codeBlock->scopeRegister()) = jsCast<JSCallee*>(callFrame->jsCallee())->scope();
```

Comment:
Why is this in common slow paths. AFAICT, it's only used in the LLInt?

Reply:
yeah, no idea either, maybe at some point it was used by baseline as well?

Reply:
Probably it was used by the baseline at some point, but it is not used anymore. So it is nice if it is moved to LLIntSlowPaths.cpp, but not strongly necessary to do it in this patch.